google-cloud/README.md

```diff
@@ -1,24 +1,20 @@
-Google Cloud Java Client
-==========================
+Google Cloud Java (All API Clients)
+===================================
 
-Java idiomatic client for [Google Cloud Platform][cloud-platform] services.
+Umbrella package that declares a dependency on all currently-available Java clients for [Google Cloud Platform][cloud-platform] services.
 
 [![Build Status](https://travis-ci.org/GoogleCloudPlatform/google-cloud-java.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/google-cloud-java)
 [![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/google-cloud-java/badge.svg?branch=master)](https://coveralls.io/r/GoogleCloudPlatform/google-cloud-java?branch=master)
-[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-core.svg)]( https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-core.svg)
+[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud.svg)]( https://img.shields.io/maven-central/v/com.google.cloud/google-cloud.svg)
 [![Codacy Badge](https://api.codacy.com/project/badge/grade/9da006ad7c3a4fe1abd142e77c003917)](https://www.codacy.com/app/mziccard/google-cloud-java)
 [![Dependency Status](https://www.versioneye.com/user/projects/58fe4c8d6ac171426c414772/badge.svg?style=flat)](https://www.versioneye.com/user/projects/58fe4c8d6ac171426c414772)
 
--  [Homepage](https://googlecloudplatform.github.io/google-cloud-java/)
--  [API Documentation](http://googlecloudplatform.github.io/google-cloud-java/apidocs)
+- See the [root README of google-cloud-java](https://googlecloudplatform.github.io/google-cloud-java/) for more details
```

Comment:
where does this link show which clients are alpha, beta, and GA?

Reply:
Good catch, that wasn't the link I wanted to use!